.gitignore

```diff
@@ -6,6 +6,7 @@ tools
 build
 .vscode
 docs/scripts/generated_script.sh
+.*.png.bkp
```

Comment:
Is this really something that should go in GitHub? What about backups of other kinds of files?

Reply:
Using Draw.io to save the editable png files creates automatically backup files with that extension. Should we rely on contributors to care for not adding those to the git repo or just have an entry in .gitignore for that? If the concern is for the `.*.png.bkp` extension, should we use something like `.*.bkp` instead?

Reply:
I think it is OK to have tool specific exclusions, if they are commented with an explanation.

Reply:
Ok, I will add a comment in .gitignore for that line